server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -83,7 +83,6 @@
 
 
 
-@AwaitsFix(bugUrl = "https://github.com/crate/crate/issues/10811")
```

Comment:
Maybe remove this in a separate commit. We've had the failures only on CI and weren't able to reproduce them locally, so removing it within this commit may give the impression that the changes here broke it - which wouldn't be the case.

Reply:
Ok, i will do that. i just wanted to have the test running and i seemed to work fine so far.